src/server/qgswmsserver.cpp

```diff
@@ -1974,26 +1974,32 @@ QImage* QgsWmsServer::createImage( int width, int height, bool useBbox ) const
 
   //Adapt width / height if the aspect ratio does not correspond with the BBOX.
   //Required by WMS spec. 1.3.
-  if ( useBbox )
+  QString version = mParameters.value( QStringLiteral( "VERSION" ), QStringLiteral( "1.3.0" ) );
+  if ( useBbox && version != QLatin1String( "1.1.1" ) )
   {
     bool bboxOk;
     QgsRectangle mapExtent = _parseBBOX( mParameters.value( "BBOX" ), bboxOk );
+    QString crs = mParameters.value( QStringLiteral( "CRS" ), mParameters.value( QStringLiteral( "SRS" ) ) );
+    if ( crs.compare( "CRS:84", Qt::CaseInsensitive ) == 0 )
+    {
+      crs = QString( "EPSG:4326" );
+      mapExtent.invert();
```

Comment:
no crs84 is an invertion of EPSG:4326.
This code has been copy from the copyMapSettings.